rhizome/host/bin/setup-vm

```diff
@@ -32,7 +32,6 @@ begin
   unix_user = params.fetch("unix_user")
   ssh_public_key = params.fetch("ssh_public_key")
   nics = params.fetch("nics").map { |args| VmSetup::Nic.new(*args) }.freeze
-  boot_image = params.fetch("boot_image")
```

Comment:
this commit message would benefit backreference to the commit that has rendered it obsolete. On its own, it is rather mysterious: a deletion of code that looks fundamental without replacement that is immediate adjacent to it in commit history (the latter being a special case where I think backreferences can be burdensome, because trival rebasing breaks the backreference).

 > Clover needs to track images available on each host, so we shouldn't download images implicitly in vm_setup.

I don't understand what's mutually exclusive between implied downloads and tracking images that are available; in principle the system might have been designed to read-back the result of the implied image download to update the `BootImage` records.

It may suffice to write that "DownloadBootImage" is how images are managed now because [commit X Y Z], without implying some intrinsic connection between laziness of download and the new regime.